docs/src/components/Toc.js

```diff
@@ -49,23 +49,27 @@ function throttle(func, wait) {
 }
 
 function useThrottledOnScroll(callback, delay) {
-  const throttledCallback = React.useMemo(() => throttle(callback, delay), [callback, delay]);
+  const throttledCallback = useMemo(() => throttle(callback, delay), [callback, delay]);
```

Comment:
Using hooks directly here and throughout, no material changes

Reply:
why directly?

Reply:
Thanks for changing this.